src/nodekit/NKScriptingLite/NKJSContext.swift

```diff
@@ -158,24 +148,24 @@ extension NKJSContext: NKScriptContext {
         
     }
 
-    public func serialize(object: AnyObject?) -> String {
+    public func serialize(_ object: AnyObject?) -> String {
     
         var obj: AnyObject? = object
         
         if let val = obj as? NSValue {
         
-            obj = val as? NSNumber ?? val.nonretainedObjectValue
+            obj = val as? NSNumber ?? val.nonretainedObjectValue as AnyObject?
        
         }
 
         
         if let s = obj as? String {
          
-            let d = try? NSJSONSerialization.dataWithJSONObject([s], options: NSJSONWritingOptions(rawValue: 0))
+            let d = try? JSONSerialization.data(withJSONObject: [s], options: JSONSerialization.WritingOptions(rawValue: 0))
```

Comment:
I think you can actually omit the `options` argument.  If I remember correctly it defaults to an empty array, so there's no need to specify a `rawValue` of 0.